txnkv/transaction/txn.go

```diff
@@ -763,6 +763,13 @@ func (txn *KVTxn) LockKeys(ctx context.Context, lockCtx *tikv.LockCtx, keysInput
 			checkedExistence = true
 		}
 	}
+	if assignedPrimaryKey && lockCtx.LockOnlyIfExists {
+		if len(keys) != 1 {
+			panic("When using the LockOnlyIfExists mode to lock keys, if one of the keys is selected as " +
+				"the primary key, the number of input keys must be one.")
```

Comment:
Use brief  English sentence or incomplete sentence to describe the reason of panicking. I think you can learn the expression style in our existing logs and error messages. You can adopt @sticnarf 's suggestion directly.